src/js/views/patients/sidebar/action/action-sidebar-activity-views.js

```diff
@@ -79,14 +79,11 @@ const ActivityView = View.extend({
     const editor = this.model.getEditor();
     const clinician = this.model.getClinician();
     const program = this.model.getProgram();
-    const toClinician = _.trim(`${ clinician.get('first_name') } ${ clinician.get('last_name') }`);
-    const role = editor.getRole().get('name');
-    const name = _.trim(`${ editor.get('first_name') } ${ editor.get('last_name') }`);
 
     return {
-      name,
-      role,
-      to_clinician: toClinician,
+      name: _.trim(editor.get('name')),
```

Comment:
shouldn't need the trims that's kinda legacy stuff from when on mainapp you could potentially not have one of the names